onnx/test/shape_inference_test.py

```diff
@@ -106,16 +113,24 @@ def _make_graph(
                 input_value_infos.append(
                     make_tensor_value_info("SEED_" + seed_name, proto_type, ())
                 )
-                input_value_infos.append(
-                    make_tensor_value_info(
-                        "UNKNOWN_SHAPE_" + seed_name, TensorProto.INT64, (None,)
+                if reshape_version >= 5:
+                    input_value_infos.append(
+                        make_tensor_value_info(
+                            "UNKNOWN_SHAPE_" + seed_name, TensorProto.INT64, (None,)
+                        )
                     )
-                )
+                if reshape_version >= 5:
+                    # For Reshape of version >= 5, it has two inputs: data and shape.
+                    reshape_inputs = ["SEED_" + seed_name, "UNKNOWN_SHAPE_" + seed_name]
+                else:
+                    # For Reshape of Version 1, it has only one input: data.
+                    reshape_inputs = ["SEED_" + seed_name]
```

Comment:
This doesn't seem right. As the comment above imply, the goal is to create a tensor of unknown-shape (to test the shape-inference logic). And a Reshape with shape as attribute (i.e., before version 5) cannot do that. We may need an alternative way to create a tensor of unknown-shape in version < 5.